src/Equinox.Cosmos/Cosmos.fs

```diff
@@ -78,7 +72,43 @@ type [<NoEquality; NoComparison; JsonObject(ItemRequired=Required.Always)>]
     /// As one cannot sort by the implicit `id` field, we have an indexed `i` field for sort and range query use
     static member internal IndexedFields = [Batch.PartitionKeyField; "i"; "n"]
 
+/// Manages zipping of the UTF-8 json bytes to make the index record minimal from the perspective of the writer stored proc
```

Comment:
That folder has been moved to Equinox.Core and its namespaces changed to FsCodec.STJ. Seeing as we wouldn't want this in FsCodec, should probably leave it in place now, right?

Reply:
Correct - you're way ahead of me as usual

Reply:
perhaps to minimize diffs, move this to L130 (and use `and`)

Reply:
I actually tried to do this originally, but there seems to be some bug in the F# compiler. If I try to use `and` and move it to line 130, then I get errors when using `[<JsonCompressedBase64Converter>]` on the fields (exact error is "No constructors are available for the type 'JsonCompressedBase64ConverterAttribute'"). So that's why for now I just moved the type up and removed `and`.

Reply:
interesting; definitely not worth losing sleep over